expr/func_constantline_test.go

```diff
@@ -0,0 +1,252 @@
+package expr
+
+import (
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/grafana/metrictank/api/models"
+	"github.com/grafana/metrictank/schema"
+	"github.com/grafana/metrictank/test"
+)
+
+type ConstantLineTestCase struct {
+	name 	string
+	value 	float64
+}
+
+// array of time ranges
+// 1s, 1m, 1hr, 1day, 30days, 400days
+var timeRanges = []uint32{
```

Comment:
Maybe this and all the `for _, to := range timeRanges {` could be moved into `testConstantLineWrapper` ? Also, might be more clear to use the existing `time.Duration` constructs like `time.Second` and `time.Minute`?